src/main/java/org/aksw/iguana/cc/controller/MainController.java

```diff
@@ -28,14 +28,16 @@ public Path convert(String value) {
         @Parameter(names = {"--ignore-schema", "-is"}, description = "Do not check the schema before parsing the suite file.")
         private boolean ignoreShema = false;
 
+        @Parameter(names = {"--dry-run", "-d"}, hidden = true)
+        public static boolean dryRun = false;
+
```

Comment:
If I remember correctly you said that hidden didn't work. 
Thus, as suggested, please look in argc for `--dry-run` and `-d` instead to set the option.

Reply:
Sorry, for the confusion. What I meant was, that I don't think it was possible to hide an option with JCommander, but I seem to have overlooked the `hidden`attribute that I used there, which does exactly what was needed. :)